### PR TITLE
fix the CSSUnitValue wrong link

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for your PR! I think this is the wrong fix though.

I don't think the fragment should be "css_typed_object_model_experimental", it should just be "css_typed_object_model". The "_experimental" bit is an artifact of the use of the {{experimental_inline}} macro in the header, which we are not supposed to do. In fact this also seems just wrong since many of these interfaces are not experimental any more (for instance, this one isn't).

So the right fix would be to remove the {{experimental_inline}} macro from the header, and the {{SeeCompatTable}} macro below it, and check each interface listed to see if it is really experimental or not.

Also, the link text in the CSSUnitValue page should be "CSS Typed Object Model API".

Please let me know if you'd like to do this, I'm happy to take care of it otherwise!

**Comment by T34-active** [comment]
> Thanks for your PR! I think this is the wrong fix though.
> 
> I don't think the fragment should be "css_typed_object_model_experimental", it should just be "css_typed_object_model". The "_experimental" bit is an artifact of the use of the {{experimental_inline}} macro in the header, which we are not supposed to do. In fact this also seems just wrong since many of these interfaces are not experimental any more (for instance, this one isn't).
> 
> So the right fix would be to remove the {{experimental_inline}} macro from the header, and the {{SeeCompatTable}} macro below it, and check each interface listed to see if it is really experimental or not.
> 
> Also, the link text in the CSSUnitValue page should be "CSS Typed Object Model API".
> 
> Please let me know if you'd like to do this, I'm happy to take care of it otherwise!

I don't know where to start, can you give me some advice or references?

File: files/en-us/web/api/cssunitvalue/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ browser-compat: api.CSSUnitValue
 
 {{APIRef("CSS Typed Object Model API")}}
 
-The **`CSSUnitValue`** interface of the {{domxref('CSS_Object_Model#css_typed_object_model','','',' ')}} represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
+The **`CSSUnitValue`** interface of the [CSS Object Model](/en-US/docs/Web/API/CSS_Object_Model#css_typed_object_model_experimental) represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
```
Comment (T34-active) [inline files/en-us/web/api/cssunitvalue/index.md:10]:
```suggestion
The **`CSSUnitValue`** interface of the [CSS Typed Object Model API](/en-US/docs/Web/API/CSS_Object_Model) represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
```

**Comment by T34-active** [comment]
@wbamberg i'm update this content, you can watch this https://github.com/mdn/content/pull/33599

**Comment by T34-active** [comment]
> Thanks for your PR! I think this is the wrong fix though.
> 
> I don't think the fragment should be "css_typed_object_model_experimental", it should just be "css_typed_object_model". The "_experimental" bit is an artifact of the use of the {{experimental_inline}} macro in the header, which we are not supposed to do. In fact this also seems just wrong since many of these interfaces are not experimental any more (for instance, this one isn't).
> 
> So the right fix would be to remove the {{experimental_inline}} macro from the header, and the {{SeeCompatTable}} macro below it, and check each interface listed to see if it is really experimental or not.
> 
> Also, the link text in the CSSUnitValue page should be "CSS Typed Object Model API".
> 
> Please let me know if you'd like to do this, I'm happy to take care of it otherwise!

/cc @wbamberg i'm update this content, you can watch this https://github.com/mdn/content/pull/33599

File: files/en-us/web/api/cssunitvalue/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ browser-compat: api.CSSUnitValue
 
 {{APIRef("CSS Typed Object Model API")}}
 
-The **`CSSUnitValue`** interface of the {{domxref('CSS_Object_Model#css_typed_object_model','','',' ')}} represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
+The **`CSSUnitValue`** interface of the [CSS Typed Object Model API](/en-US/docs/Web/API/CSS_Object_Model) represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
```
Comment (wbamberg) [inline files/en-us/web/api/cssunitvalue/index.md:10]:
```suggestion
The **`CSSUnitValue`** interface of the [CSS Typed Object Model API](/en-US/docs/Web/API/CSS_Object_Model#css_typed_object_model) represents values that contain a single unit type. For example, "42px" would be represented by a `CSSNumericValue`.
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @T34-active , and thanks for fixing all the other macro calls in the other PR <3.

**Pull request merged**